### ISSUE TITLE
[Feature Request] Series Title and and Book number on title detailed information screen

### ISSUE THREAD

**Issue by ShadowPeo** [LeoKlaus/plappa#67]
Please put the series name (as a link to the series) and the book number on the detailed information page (page where you see the description of the book) I utilise this all the time in both Audible and the TestFlight AudioBookshelf app to get the next book in the series playing so I can go straight to the next book when one finishes

**Comment by ElysiumFic** [comment]
I just started investigating using plappa/Jellyfin as an alternative to Prologue/Plex, and this is quite distressing, since lack of series support is my top source of discontent with just about any audiobook player option.

PLEASE implement series support (bonus points if you figure out a solution for situations where you have multiple series within a shared universe.)

**Comment by LeoKlaus** [comment]
> PLEASE implement series support (bonus points if you figure out a solution for situations where you have multiple series within a shared universe.)

There's already series support with AudioBookShelf (it's just not displayed in the detail view as of now). Books can be part of multiple series there as well, so multiple series in a shared universe shouldn't be an issue.

As far as I know, Jellyfin doesn't provide any information on series, so there's no way for plappa to handle series with Jellyfin (correct me if I'm wrong).

I'd really suggest you try ABS for audiobooks, it's a lot better than Jellyfin for that. You should be able to just point ABS at your Jellyfin media folder.

**Comment by ElysiumFic** [comment]
Thank you, I checked out ABS a couple years ago and they didn't have series support then, and my seedbox didn't have support for it at that time (and still doesn't as far as I know) so it looks like I need to revisit that idea. Thanks so much.

________________________________________

**Comment by ShadowPeo** [comment]
Yes ABS is the way to go for this, I have switched to this totally and very happy with ABS. Both players on iOS have their issues but Plappa is the better of the two (hence I paid for it) but this is just one small gripe of mine. 

Podcasts is another thing but I think those are ABS issues not supporting what I want, not Plappa but I have not investigated
